### PR TITLE
Updated prefix list

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
Ready

## Related Issues
https://github.com/demisto/demisto-sdk/pull/2506

## Description
I've updated the prefix list adding the newly approved ones.

### CODE REVIEW

File: docs/documentation/release-notes.md [hunk 1]
```diff
@@ -121,7 +121,12 @@ Specify in the corresponding release notes file:
 'Updated the Docker image to '
```
Comment (idovandijk) [inline docs/documentation/release-notes.md:121]: If we have "Updated the" we don't need this one right?

**Review by idovandijk — COMMENTED** [review]
comment

File: docs/documentation/release-notes.md (hunk 1 above)
Comment (Ni-Knight) [reply docs/documentation/release-notes.md:121]: guess so, but this is just the documentation, do you want to remove it from here?

Comment (idovandijk) [reply docs/documentation/release-notes.md:121]: Oh, I don't mind it then, your call

**Review by idovandijk — APPROVED** [review]

File: docs/documentation/release-notes.md (hunk 1 above)
Comment (Ni-Knight) [reply docs/documentation/release-notes.md]: Made the change, so it wont look weird

**Pull request merged**